packages/compass-databases-navigation/src/collection-item.tsx

```diff
@@ -38,6 +38,7 @@ const CollectionIcon: React.FunctionComponent<{
 
 const collectionItem = css({
   height: COLLECTION_ROW_HEIGHT,
+  paddingLeft: spacing[4],
```

Comment:
This probably is also needed for the placeholder when the type is collection

Reply:
We should probably restructure the code/css so we don't have to duplicate these things.